RIOT-OS/RIOT
# ci/build_system_sanity_check: ensure LINKFLAGS/LINKER_SCRIPT are not exported

### Contribution description

This PR add a static test to ensure LINKFLAGS and LINKER_SCRIPT variables are not exported anymore.




### Testing procedure

It should fail in the current state (e.g. without #13017 merged into master).

Otherwise, a green Murdock should be enough (eventually with run-test enabled).
One could also check that the size of a firmware build on an ARM platform doesn't change.




### Issues/PRs references

Can't be merged before #13017 is merged.
